src/core_modules/capture-core/utils/navigateToTrackedEntityDashboard.js

```diff
@@ -1,7 +1,11 @@
 import { config } from 'd2';
 
-export const navigateToTrackedEntityDashboard = (teiId, orgUnitId, scopeSearchParam) => {
+export const navigateToTrackedEntityDashboard = (teiId, orgUnitId, scopeSearchParam, currentUrl) => {
     const { baseUrl } = config;
     const instanceBaseUrl = baseUrl.split('/api')[0];
-    window.location.href = `${instanceBaseUrl}/dhis-web-tracker-capture/#/dashboard?tei=${teiId}&ou=${orgUnitId}&${scopeSearchParam}`;
+    const base64Url = btoa(`/capture-app/index.html#${currentUrl}`);
```

Comment:
The urls will be of the following format `/capture-app/index.html#${currentUrl}` which basically will be something like `/capture-app/index.html#/new?programId=ID&orgUnitId=ID` or `/capture-app/index.html#/search?trackedEntityType=IDorgUnitId=ID` and so on

Reply:
I have a suggestion :) Can the `returnUrl` be the whole url except the `baseUrl`? That way we would avoid any potential problems if the app name changes (or if we're uploading a custom app).

Reply:
uhuh surely can do that

Reply:
Alternatively we need to change `capture-app` in the url to `dhis-web-capture`. But I would prefer the above.

Reply:
Help me understand this right, we want to omit the first bit of this `const base64Url = btoa(`/capture-app/index.html#${currentUrl}`);` and have as a result this ->`const base64Url = btoa(`/index.html#${currentUrl}`);`?

Reply:
Problem is I dont know bu heart what value the `baseUrl` holds usually..

Reply:
On line 4 in the current file you have: `const { baseUrl } = config;`

The return url I would like to be the entire url in the address bar, except the baseUrl (from line 4). 

Does that make sense?

Reply:
Almost. Can you give an example please?

Reply:
Okay now got it. Makes sense

Reply:
On second thought, it won't work as I imagined. Let's go with the easy solution.